tests/wp-unit/include/Core/Metas/Modules/DeleteTermMetaModuleTest.php

```diff
@@ -0,0 +1,149 @@
+<?php
+
+namespace BulkWP\BulkDelete\Core\Metas\Modules;
+
+use BulkWP\Tests\WPCore\WPCoreUnitTestCase;
+
+/**
+ * Test Delete Term Meta Module.
+ *
+ * Tests \BulkWP\BulkDelete\Core\Metas\Modules\DeleteTermMetaModule
+ *
+ * @since 6.0.0
+ */
+class DeleteTermMetaModuleTest extends WPCoreUnitTestCase {
+
+	/**
+	 * The module that is getting tested.
+	 *
+	 * @var \BulkWP\BulkDelete\Core\Metas\Modules\DeleteTermMetaModule
+	 */
+	protected $module;
+
+	/**
+	 * Setup the Module.
+	 */
+	public function setUp() {
+		parent::setUp();
+
+		$this->module = new DeleteTermMetaModule();
+	}
+
+	/**
+	 * Add to test delete default taxonomy term meta with equal value.
+	 */
+	public function test_that_delete_default_taxonomy_term_meta_with_equal_value() {
```

Comment:
Function name is not following the `test_that_something_can_be_deleted` format.

Change it for all the methods.

Reply:
Fixed